terracumber_config/tf_files/Uyuni-Master-NUE.tf

```diff
@@ -149,8 +149,8 @@ module "cucumber_testsuite" {
       install_salt_bundle = true
     }
     suse-minion = {
-      image = "sles15sp4o"
-      name = "min-sles15"
+      image = "opensuse154o"
+      name = "min-leap15"
```

Comment:
In order to be aligned with my suggestion on domain names, we should be using "min-suse" here.

Reply:
Agreed after discussing this with some more team members in Slack: https://suse.slack.com/archives/C02D12TNYLS/p1691050880162189